src/components/TeamTab.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import {Accordion, AccordionSummary, AccordionDetails} from '@material-ui/core';
+import TeamBio from '../components/TeamBio';
+import headshot1 from '../../assets/img/headshot1.png'
+import headshot2 from '../../assets/img/headshot2.png'
+import headshot3 from '../../assets/img/headshot3.png'
+import headshot4 from '../../assets/img/headshot4.png'
+
+
+export default function TeamTab(props) {
+  const {expanded, handleChange, teamName} = props;
+
+  return (
+      <Accordion className="tab-main" expanded={expanded} onChange={handleChange}>
+        <AccordionSummary className="tab-header" aria-controls="panel1d-content" id="panel1d-header">
+          <h1 className="team-title">{teamName}</h1>
+        </AccordionSummary>
+        <AccordionDetails className="tab-body">
+          <TeamBio imgSrc={headshot1} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot2} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot3} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot4} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+
+          <TeamBio imgSrc={headshot1} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot2} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot3} name={'Mimi Jiao'} title={'Co-Head Director'}/>
+          <TeamBio imgSrc={headshot4} name={'Mimi Jiao'} title={'Co-Head Director'}/>
```

Comment:
I know you're still developing, but is it possible to make arrays of teams so that you can generate this with a for loop?